homework-submission/week1-project/src/Components/todoComponents.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+import React, { Component } from 'react';
+
+class Description extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            onEdit: false,
+        };
+      }
+    
+    handleEdit = (e) => {
+        if(e.target.name === 'edit')
+            this.setState({onEdit: true});
+        else
+        this.setState({onEdit: false});
+    
+    }
+    handleChange = (e) => {
+        if(e.target.name === 'description')
+            this.props.editTodo(this.props.id, e.target.value, this.props.deadline);
+        else
+        this.props.editTodo(this.props.id, this.props.description,  e.target.value);
```

Comment:
What about
```suggestion
        this.props.editTodo(this.props.id, this.props[e.target.name],  e.target.value);
```
and removing the previous lines?